openfisca_france_dotations_locales/variables/potentiel_financier.py

```diff
@@ -44,4 +44,5 @@ class potentiel_financier_par_habitant(Variable):
     definition_period = YEAR
 
     def formula(commune, period, parameters):
-        return commune('potentiel_financier', period) / commune('population_dgf', period)
+        population_dgf = commune('population_dgf', period)
+        return (population_dgf > 0) * np.divide(commune('potentiel_financier', period), population_dgf, where = (population_dgf > 0))
```

Comment:
De même, ne pas indiquer la condition deux fois ? 🤔 
```suggestion
        return np.divide(commune('potentiel_financier', period), population_dgf, where = (population_dgf > 0))
```